simtools/configuration/commandline_parser.py

```diff
@@ -197,20 +197,20 @@ def initialize_telescope_model_arguments(self, add_model_version=True, add_teles
 
         Parameters
         ----------
-        add_model_version: str
-            Model version.
-        add_telescope: str
-            Telescope model name (e.g. LST-1, SST-D, ...).
+        add_model_version: bool
+            Set if to allow a simulation model argument.
+        add_telescope: bool
+            Set if to allow a telescope name argument.
         """
 
         _job_group = self.add_argument_group("telescope model")
         _job_group.add_argument(
-            "--site", help="CTAO site (e.g. North, South)", type=self.site, required=False
+            "--site", help="CTAO site (e.g., North, South)", type=self.site, required=False
```

Comment:
in `simulate_prod` we are using paranal and la palma for the sites, and not North/South. Not sure if this is an issue, but we should be consistent.

Reply:
I actually switched to use the default configurator option instead of redefining it in the application. I was a bit hesitant originally cause I was worried about having both `--site north --from_azimuth_direction north` since it might be a bit confusing. However, I guess for the expected users of this application it should be very clear.
I was also not too happy about defining the `--telescope` option unnecessarily which the default configurator does, but it's not a real issue.